spring-cloud-gcp-samples/pom.xml

```diff
@@ -79,7 +79,8 @@
 							<dependency>
 								<groupId>io.spring.javaformat</groupId>
 								<artifactId>spring-javaformat-checkstyle</artifactId>
-								<version>0.0.9</version>
+                                <!-- Keep in sync with spring-javaformat-checkstyle.version in pom.xml -->
+                                <version>0.0.25</version>
```

Comment:
I think we need tabs here. :)

Reply:
sheesh. Checkstyle could have told me, ya know?